modules/command/dv/dv-purchase-dispute-command.js

```diff
@@ -18,8 +24,97 @@ class DvPurchaseDisputeCommand extends Command {
      */
     async execute(command, transaction) {
         // send dispute purchase to bc
+        const {
+            handler_id,
+            encoded_data,
+            key,
+            purchase_id,
+            permissioned_data_array_length,
+            permissioned_data_original_length,
+            error_type,
+        } = command.data;
+
+        try {
+            const purchaseStatus = await this.blockchain.getPurchaseStatus(purchase_id);
+
+            if (purchaseStatus !== '2') {
+                throw new Error(`Cannot issue complaint for purchaseId ${purchase_id}. Purchase already completed`);
+            }
+
+            this.remoteControl.purchaseStatus('Purchase not confirmed', 'Sending dispute purchase to Blockchain.', true);
+
+            this.logger.important(`Initiating complaint for purchaseId ${purchase_id}`);
+
+            let result;
+            if (error_type === constants.PURCHASE_ERROR_TYPE.node_error) {
```

Comment:
It is more consistent to use uppercase for constants

Reply:
Yeah, it does, I'll update it asap 😄